runtime/polkadot/tests/weights.rs

```diff
@@ -57,31 +56,6 @@ fn weight_of_system_set_code_is_correct() {
 	assert_eq!(weight, expected_weight);
 }
 
-#[test]
```

Comment:
We remove these tests because the weights no longer come from downstream Substrate, but we define them locally in the Repo.